scripts/config/lib/ngx_mruby/routes_path.rb

```diff
@@ -0,0 +1,33 @@
+# ghetto require, since mruby doesn't have require
+eval(File.read('/app/bin/config/lib/nginx_config_util.rb'))
+
+USER_CONFIG = "/app/static.json"
+
+def match_excepts(excepts, uri)
+  return false unless excepts
+
+  excepts.each do |except|
+    return true if Regexp.compile("^#{except}$") =~ uri
```

Comment:
Given that every single call site bolts `^` and `$` onto the regex, should we have `to_regex` set those up, instead?